client/src/pages/learn/coding-interview-prep/take-home-projects/index.md

```diff
@@ -5,6 +5,13 @@ superBlock: Coding Interview Prep
 ---
 ## Introduction to the Coding Interview Take Home Projects
 
-When applying for a job as a developer, there's a good chance the hiring team will give you something called a "take-home project". As the name suggests, it's something that you're supposed to do on your own, without the interviewer looking over your shoulder. Unlike other parts of the coding interview, which often focus more on theoretical knowledge and less on actual app development - a take-home project is usually the best place to show your skills in practice.
+Programming interviews have always been stressful. One of the solutions is giving the applicant a take home project to be completed outside of the interview. These types of interviews usually require a lot of work, but it is the most preferred interviewing process.
```

Comment:
Most preferred interviewing process for whom?  The interviewer or interviewee?   Can you provide evidence for this? The previous verbiage was simpler to understand and meant something different.

Reply:
For both the Applicant and Technical HR @cmccormack

Reply:
@Sumer16 As @cmccormack mentioned, we need a reference linked which states this - 

> but it is the most preferred interviewing process.